src/property-dictionary.js

```diff
@@ -20,7 +20,8 @@
      'pitch-range': [NumberValue],
      'border-top-width': [LengthValue],
      'opacity': [NumberValue],
-     'animation-iteration-count': [NumberValue]
+     'animation-iteration-count': [NumberValue],
+     'transform': [TransformValue]
```

Comment:
It'd be good to get these in alphabetical order sooner rather than later. Same below.